src/analytics/src/main/scala/osmesa/analytics/Footprints.scala

```diff
@@ -325,11 +320,12 @@ object Footprints extends Logging {
           RasterWithSequenceTileSeqWithTileCoordinatesAndKey(
             tiles
               .map(x => RasterWithSequence(x.raster, x.sequence))
-              .toSeq,
+              .toVector, // this CANNOT be toSeq, as that produces a stream, which is subsequently partially consumed
```

Comment:
**This** is the fix for dropped sequence tiles. I'm still unclear where it was being partially consumed (I was thinking `tileSeq.tiles.isEmpty`, above, but apparently not).